MAUI/Themes/Keys.md

```diff
@@ -107,3 +107,2093 @@ This page lists the keys for each control and the element to which it is mapped
+
+	<tr>
+    <td>
+            SfImageEditorNormalToolbarIconColor       
+            <br/>
+            <br/>
+        </td>
+        <td>
+            Color of toolbar icon background in image editor. 
+            <br/>
+            <br/>
+        </td>
+    </tr>
+
+	<tr>
+    <td>
+            SfImageEditorDisabledToolbarIconColor        
+            <br/>
+            <br/>
+        </td>
+        <td>
+            Color of toolbar icon background in image editor when it is disabled.  
+            <br/>
+            <br/>
+        </td>
+    </tr>
+
+	<tr>
+    <td>
+            SfImageEditorHoverToolbarIconBackground         
+            <br/>
+            <br/>
+        </td>
+        <td>
+            Color of toolbar icon background in image editor when it is hovered.   
+            <br/>
+            <br/>
+        </td>
+    </tr>
+
+	<tr>
+    <td>
+            SfImageEditorPressedToolbarIconBackground          
+            <br/>
+            <br/>
+        </td>
+        <td>
+            Color of toolbar icon background in image editor when it is in pressed state. 
+            <br/>
+            <br/>
+        </td>
+    </tr>
+
+	<tr>
+    <td>
+            SfImageEditorNormalToolbarTextColor           
+            <br/>
+            <br/>
+        </td>
+        <td>
+            Color of toolbar text in image editor.   
+            <br/>
+            <br/>
+        </td>
+    </tr>
+
+	<tr>
+    <td>
+            SfImageEditorHoverToolbarTextColor            
+            <br/>
+            <br/>
+        </td>
+        <td>
+            Color of toolbar text in image editor when it is hovered.   
+            <br/>
+            <br/>
+        </td>
+    </tr>
+
+	<tr>
+    <td>
+            SfImageEditorPressedToolbarTextColor             
+            <br/>
+            <br/>
+        </td>
+        <td>
+            Color of toolbar text in image editor when it is in pressed state.    
+            <br/>
+            <br/>
+        </td>
+    </tr>
+
+	<tr>
+    <td>
+            SfImageEditorDisabledToolbarTextColor              
+            <br/>
+            <br/>
+        </td>
+        <td>
+            Color of toolbar text in image editor when it is disabled.    
+            <br/>
+            <br/>
+        </td>
+    </tr>
+
+	<tr>
+    <td>
+            SfImageEditorTooltipBackground               
+            <br/>
+            <br/>
+        </td>
+        <td>
+            Color of tooltip background in image editor.     
+            <br/>
+            <br/>
+        </td>
+    </tr>
+
+	<tr>
+    <td>
+            SfImageEditorTooltipTextColor                
+            <br/>
+            <br/>
+        </td>
+        <td>
+            Color of tooltip text in image editor.      
+            <br/>
+            <br/>
+        </td>
+    </tr>
+
+	<tr>
+    <td>
+            SfImageEditorFocusedColorPaletteColor                 
+            <br/>
+            <br/>
+        </td>
+        <td>
```

Comment:
Color of color palate icon when it is selected.

Reply:
Updated